test/e2e/scalability/density.go

```diff
@@ -442,6 +440,7 @@ var _ = SIGDescribe("Density", func() {
 
 		uuid = string(utiluuid.NewUUID())
 
+		framework.ExpectNoError(framework.ResetSchedulerLatencyMetrics(c))
```

Comment:
Actually, let's name this function as `ResetSchedulerMetrics` (as we may expand it in future to reset more metrics than just latency).